docs/How_To_Update_ONNX_Dev_Notes.md

```diff
@@ -15,18 +15,12 @@ git add onnx
 2. Update [cgmanifests/submodules/cgmanifest.json](/cgmanifests/submodules/cgmanifest.json).
 This file should be generated. See [cgmanifests/README](/cgmanifests/README.md) for instructions.
 
-3. Update [tools/ci_build/github/linux/docker/scripts/install_onnx.sh](/tools/ci_build/github/linux/docker/scripts/install_onnx.sh).
-Search 'for version2tag', update the commit hashes. The list should contain every release version from ONNX 1.2, and the latest one in our cmake/external/onnx folder.
+3. Update the commit id of onnx for building the linux image for CI pipelines
+- [manylinux image requirements](onnxruntime/tools/ci_build/github/linux/docker/scripts/manylinux/requirements.txt)
```

Comment:
>manylinux image requirement [](start = 3, length = 27)

Are the images generated automatically once those files are updated? Do we have to notify anyone?

Reply:
After #5906 get merged, they will be fully automated, so that there is no need to manually generate the docker images.